src/zzz_od/gui/view/home/home_interface.py

```
@@ -637,23 +707,33 @@ def reload_banner(self, show_notification: bool = False) -> None:
-        elif self.ctx.custom_config.remote_banner and os.path.exists(remote_banner_path):
-            banner_path = remote_banner_path
+        custom_banner_dir = Path(os_utils.get_path_under_work_dir('custom', 'assets', 'ui')) / 'banner'
+        ui_dir = Path(os_utils.get_path_under_work_dir('assets', 'ui'))
+        official_dynamic_path = ui_dir / 'official_dynamic.webm'
+        version_poster_path = ui_dir / 'version_poster.webp'
+        remote_banner_path = ui_dir / 'remote_banner.webp'
+        index_banner_path = ui_dir / 'index.png'
+
+        # 主页背景优先级：自定义 > 枚举选项 > index.png
+        if self.ctx.custom_config.custom_banner:
+            # 检测自定义背景文件（支持图片和视频）
```

Ruff (0.14.0)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)